frontend/src/components/Router.tsx

```diff
@@ -58,7 +69,7 @@ export const RoutePage = {
   NEW_EXPERIMENT: '/experiments/new',
   NEW_RUN: '/runs/new',
   PIPELINES: '/pipelines',
-  PIPELINE_DETAILS: `/pipelines/details/:${RouteParams.pipelineId}`,
+  PIPELINE_DETAILS: `/pipelines/details/:${RouteParams.pipelineId}?`, // pipelineId is optional
```

Comment:
This `?` is correct? shouldn't it just be added later if there are query params?

Reply:
This is the route path, the current location page is matched against all of these paths to see which one matches, so it needs to include the `?`.